src/components/blipSelect/CreatableOptionsList.js

```diff
@@ -97,9 +100,10 @@ export class CreatebleOptionsList extends Component {
    * Render add new option
    */
   _renderAddOption(newOption) {
-    const addOptionHtml = this.options.addOptionText
-      ? html`<span class="blip-prompt-add-option">${this.options.addOptionText}</span>`
-      : ''
+    const addOptionHtml = this.options.addOptionText ? html`
+      <span class="blip-prompt-add-option">
+        ${this.options.addOptionText}
+      </span>` : ''
```

Comment:
Ideally, you shouldn't have a variable that can take different types. It isn't a problem in this context but it's not a good practice